README.md

```diff
@@ -12,63 +12,159 @@ purposes:
 See [Echoes documentation](https://docs.echoeshq.com/gitlab-api-proxy) for more
 information.
 
-## API keys
+## Quickstart
 
-The setting of the API keys is done in the `api_keys.conf` file. This file
-should be mounted dynamically and not baked into the docker image.
+### Step 1: Clone the repository
 
-## API routes
+```console
+git clone git@github.com:EchoesHQ/gitlab-proxy.git
+```
 
-Allowed endpoints and methods are defined in `gitlab-api-routes` folder.
+### Step 2: Build the Docker image
 
-## HTTP headers
+Navigate to the `nginx-proxy` folder.
```

Comment:
There are inconsistencies in the document whether the instructions end with `.`, `:`, or nothing. I'd vote for `:` but I don't really care as long as it's consistent :-)